board/safety/safety_subaru.h

```diff
@@ -14,6 +14,16 @@ const SteeringLimits SUBARU_STEERING_LIMITS = SUBARU_STEERING_LIMITS_GENERATOR(2
 const SteeringLimits SUBARU_GEN2_STEERING_LIMITS = SUBARU_STEERING_LIMITS_GENERATOR(1000, 40, 40);
 
 
+const LongitudinalLimits SUBARU_LONG_LIMITS = {
+  .min_gas = 808,       // appears to be engine braking
+  .max_gas = 3400,      // approx  2 m/s^2 when maxing cruise_rpm and cruise_throttle
+  .inactive_gas = 1818, // this is zero acceleration
+  .max_brake = 600,     // approx -3.5 m/s^2
+
+  .min_alt = 0,         // Transmission RPM limits
```

Comment:
this alt name is confusing

Reply:
the goal was to make it generic in case any other brands have a similar "alternative" gas or brake signal that needs limiting. "rpm" is also weird, because it's actually the transmission rpm not the engine rpm

Reply:
it's **incredibly important** that this particular code is as simple and readable as possible. in this case, the variable should probably be called `min_transmission_rpm`

Reply:
renamed to transmission_rpm